gammapy/utils/fitting/fit.py

```diff
@@ -332,22 +338,33 @@ def likelihood_contour(self):
         """
         raise NotImplementedError
 
-    def minos_contour(self):
+    def minos_contour(self, x, y, numpoints=10, sigma=1.0):
         """Compute MINOS contour.
 
```

Comment:
Maybe add a `Parameters` section to the docstring? As the signature is the same as `Minuit.mncontour`, it might not be needed...

Reply:
OK, I'll add it.